share/sysbuild/image_configurations/MAIN_image_default.cmake

```diff
@@ -33,7 +33,10 @@ if(SB_CONFIG_BOOTLOADER_MCUBOOT)
   elseif(SB_CONFIG_MCUBOOT_MODE_DIRECT_XIP_WITH_REVERT)
     set_config_bool(${ZCMAKE_APPLICATION} CONFIG_MCUBOOT_BOOTLOADER_MODE_DIRECT_XIP_WITH_REVERT y)
   elseif(SB_CONFIG_MCUBOOT_MODE_RAM_LOAD)
-    # Not yet supported in zephyr code
+    # RAM load mode requires XIP be disabled and flash size be set to 0
+    set_config_bool(${ZCMAKE_APPLICATION} CONFIG_MCUBOOT_BOOTLOADER_MODE_RAM_LOAD y)
+    set_config_bool(${ZCMAKE_APPLICATION} CONFIG_XIP n)
+    set_property(TARGET ${ZCMAKE_APPLICATION} APPEND_STRING PROPERTY CONFIG "CONFIG_FLASH_SIZE=0\n")
```

Comment:
Can we use `set_config_string`?

Reply:
No, this is not a string, it is an integer